tests/test-ssh-remote-command.cmd

```diff
@@ -1,4 +1,4 @@
-footloose config create --config %t.footloose --name %t --key %t-key
-footloose create --config %t.footloose
-%out footloose --config %t.footloose ssh node0 hostname
-footloose delete --config %t.footloose
+../footloose config create --config %t.footloose --name %t --key %t-key
```

Comment:
the tests expert `footloose` to be in the path, don't think we need to change that, maye using `go install` would help here. Something like:

```
go install # in the root of this repository
export PATH=`go env GOPATH`/bin:$PATH
```